src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -41,9 +43,17 @@ export default (teamId: string) => {
   }
   const team = teamId ? data?.me?.teams.find(team => team.id === teamId) : data?.me?.myTeam;
 
-  const { data: projectData } = useGetProjectsQuery({
-    variables: { teamId: teamId ?? "", first: 100 },
+  const initProjectPerPage = 5;
+  const projectPerPage = 3;
+  const {
+    data: projectData,
+    loading: queryProject,
```

Comment:
With the other paginated queries, we just do `loading`. Can you keep it same as elsewhere?

Reply:
it is using from other query useGetMequery

Reply:
Ok, if you can rename it to `loadingProjects` I think it reflects its meaning better.

Reply:
This wasn't resolved

Reply:
sorry I renamed the returned value as it was projectLoading , I thought it is the one I should renamed it , done